Assets/Scripts/Editor/BeatMappingWindow.cs

```diff
@@ -0,0 +1,195 @@
+using ProefExamen.Audio.WaveFormDrawer;
+using ProefExamen.Audio.SpectrumDrawer;
+using ProefExamen.Audio.TimeStamping;
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+using UnityEditor.UIElements;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
```

Comment:
```suggestion
using System.Collections.Generic;
using UnityEngine.UIElements;
using UnityEditor.UIElements;
using System.Linq;
using UnityEditor;
using UnityEngine;

using ProefExamen.Audio.WaveFormDrawer;
using ProefExamen.Audio.SpectrumDrawer;
using ProefExamen.Audio.TimeStamping;
```
According to code conventions.